fineract-provider/src/main/java/org/apache/fineract/portfolio/loanaccount/api/LoansApiResourceSwagger.java

```diff
@@ -1364,7 +1364,7 @@ private PutLoansLoanIdDisbursementData() {}
             @Schema(example = "1")
             public BigDecimal netDisbursalAmount;
             @Schema(example = "1")
-            public Integer interestType;
+            public Long interestType;
```

Comment:
This looks wrong to me based on it is the ordinal of an enum.

Reply:
Done.